README.md

```diff
@@ -175,3 +175,36 @@ const binary = packet.serializeIlpFulfillment({
 
 const json = packet.deserializeIlpFulfillment(binary)
 ```
+
+### IlpPrepare, IlpFulfill, IlpReject
+
+```js
+const packet = require('ilp-packet')
+const crypto = require('crypto')
+function sha256 (preimage) { return crypto.createHash('sha256').update(preimage).digest() }
+
+const fulfillment = crypto.randomBytes(32)
+const condition = sha256(fulfillment)
+
+const binaryPrepare = packet.serializeIlpPrepare({
+  amount: '10',
+  executionCondition: condition,
+  destination: 'g.us.nexus.bob', // this field was called 'account' in older packet types
+  data: Buffer.from(['hello world']),
+  expiresAt: new Date(new Date().getTime() + 10000)
+})
+
+const binaryFulfill = packet.serializeIlpFulfill({
+  fulfillment,
+  data: Buffer.from('thank you')
+})
+
+// not to be confused with IlpRejection:
```

Comment:
So we have two different objects named IlpRejection and IlpReject? That is definitely confusing.

Reply:
Yes. Especially because both use the same TypeScript interface, see https://github.com/interledgerjs/ilp-packet/blob/master/index.ts#L713 and https://github.com/interledgerjs/ilp-packet/blob/master/index.ts#L735

Reply:
It will become simpler because all packet types up to 11 will soon be marked as deprecated, so then 'ilp rejection' will disappear and only 'ilp reject' will survive.